src/cc_toolchain/BUILD

```diff
@@ -23,6 +23,21 @@ environment_group(
     ],
 )
 
+config_setting(
+    name = "cpu_k8",
+    values = {"cpu": "k8"},
+)
+
+config_setting(
+    name = "cpu_stm32f4",
+    values = {"cpu": "stm32f4"},
+)
+
+config_setting(
+    name = "cpu_stm32h7",
+    values = {"cpu": "stm32h7"},
+)
```

Comment:
Unfortunately we need these to use the `select` in `firmware/main/app/primitives/BUILD`, we can't just use the `cpu` from the environment directly.